### PR TITLE
more logging

### PR DESCRIPTION
**Pull request by enola-dkfz**

### CODE REVIEW

**Review by Threated — APPROVED** [review]

File: src/main.rs [hunk 1]
```diff
@@ -268,6 +268,8 @@ async fn run_cql_query(
 
             let cql_result = blaze::run_cql_query(&query.lib, &query.measure).await?;
 
+            trace!("MeasureReport with unobfuscated values: {}", &cql_result);
```
Comment (TKussel) [inline src/main.rs:271]: As the trace level now logs unobfuscated results, I would like to see a fat warning during startup, if `trace` is selected.

**Review by TKussel — CHANGES_REQUESTED** [review]
Looks great, however, please add the warning during startup if trace log level is selected.

File: src/main.rs (hunk 1 above)
Comment (enola-dkfz) [reply src/main.rs:271]: made a new PR into develop with that

**Review by TKussel — APPROVED** [review]

**Comment by TKussel** [comment]
The comment is addressed in #157

**Pull request merged**